src/plugins/console/server/lib/spec_definitions/js/aggregations.ts

```diff
@@ -113,6 +113,8 @@ const rules = {
     diversified_sampler: {
       shard_size: '',
       field: '',
+      max_docs_per_value: '',
```

Comment:
Q: I see in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-diversified-sampler-aggregation.html#_max_docs_per_value) that the default is `1`, shouldn't it be reflected here also?

Reply:
Fixed